plugins/org.jkiss.dbeaver.tasks.ui.view/src/org/jkiss/dbeaver/tasks/ui/view/DatabaseTasksView.java

```diff
@@ -359,13 +371,17 @@ public void handleTaskFolderEvent(DBTTaskFolderEvent event) {
     }
 
     private void loadViewConfig() {
-        tasksTree.loadViewConfig();
+        if (tasksTree != null) {
+            tasksTree.loadViewConfig();
+        }
```

Comment:
A nitpick: other changes use an early return technique, perhaps it can be used here too?

```suggestion
        if (tasksTree == null) {
            return;
        }
        tasksTree.loadViewConfig();
```